src/interface/metadata.cpp

```diff
@@ -128,7 +128,7 @@ std::array<int, 6> Metadata::GetArrayDims(std::weak_ptr<MeshBlock> wpmb,
     // classes add the +1's where needed.  They all expect
     // these dimensions to be the number of cells in each
     // direction, NOT the size of the arrays
-    assert(N >= 1 && N <= 3);
+    assert(N >= 0 && N <= 3);
```

Comment:
Not sure I follow this change

Reply:
This is to accommodate scalar fields having `shape = {}` as above, which means `N = 0` here.